packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -10,123 +10,69 @@
 import {SpectrumFieldProps} from '@react-types/label';
 import {useFormProps} from '@react-spectrum/form';
 import {useMessageFormatter} from '@react-aria/i18n';
+import {useStyleProps} from '@react-spectrum/utils';
 
 function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
   props = useFormProps(props);
   let {
     label,
-    labelPosition = 'top' as LabelPosition,
-    labelAlign,
-    isRequired,
-    necessityIndicator,
-    includeNecessityIndicatorInAccessibilityName,
     validationState,
     description,
     errorMessage,
-    isDisabled,
-    showErrorIcon,
     children,
-    labelProps,
     readOnlyText,
     isReadOnly,
-    inputProps, 
-    inputRef, 
-    // Not every component that uses <Field> supports help text.
-    descriptionProps = {},
-    errorMessageProps = {},
-    elementType,
-    wrapperClassName,
+    inputRef,
 
     ...otherProps
   } = props;
   let {styleProps} = useStyleProps(otherProps);
   let hasHelpText = !!description || errorMessage && validationState === 'invalid';
   let formatMessage = useMessageFormatter(intlMessages);
   let displayReadOnly = isReadOnly && (readOnlyText || readOnlyText === '');
+  let defaultInputRef = useRef<HTMLTextAreaElement>(null);
+  inputRef = inputRef || defaultInputRef;
 
-  if (label || hasHelpText) {
-    let labelWrapperClass = classNames(
-      labelStyles,
-      'spectrum-Field',
-      {
-        'spectrum-Field--positionTop': labelPosition === 'top',
-        'spectrum-Field--positionSide': labelPosition === 'side'
-      },
-      styleProps.className,
-      wrapperClassName
-    );
-
-    children = React.cloneElement(children, mergeProps(children.props, {
-      className: classNames(
-        labelStyles,
-        'spectrum-Field-field'
-      )
-    }));
-
-    let renderHelpText = () => (
-      <HelpText
-        descriptionProps={descriptionProps}
-        errorMessageProps={errorMessageProps}
-        description={description}
-        errorMessage={errorMessage}
-        validationState={validationState}
-        isDisabled={isDisabled}
-        showErrorIcon={showErrorIcon} />
+  if (displayReadOnly) {
+    if (readOnlyText === '') {
+      readOnlyText = formatMessage('(None)');
+    }
+    
+    children = (
+      <ReadOnlyField
+        {...props} 
+        readOnlyText={readOnlyText}
+        displayReadOnly={displayReadOnly}
+        hasHelpText={hasHelpText}
+        styleProps={styleProps}
+        inputRef={inputRef as RefObject<HTMLTextAreaElement>} 
+        ref={ref as RefObject<HTMLDivElement>} />
     );
 
-    if (displayReadOnly) {
-      if (readOnlyText === '') {
-        readOnlyText = formatMessage('(None)');
-      }
-      children = (
-        <ReadOnlyField
-          {...props} 
-          readOnlyText={readOnlyText}
-          inputProps={inputProps} 
-          ref={inputRef as RefObject<HTMLTextAreaElement>} />            
-      );
+    if (label) {
+      return children;
     }
```

Comment:
As a second step after the tests are fully fleshed out:
- remove this if clause
- replace cloneElement in ReadOnlyField with `return textfield`;

Theoretically this should handle unsafe_classname/style/ref merging for us still, but double check with tests and storybook

Reply:
will probably allow us to get rid of some props from ReadOnlyField